src/domain/index.ts

```diff
@@ -63,15 +70,21 @@ export function init(repositories: Repositories, appConfig: AppConfig): DomainSe
   );
 
   const editorToolsService = new EditorToolsService(repositories.editorToolsRepository);
-  const userService = new UserService(repositories.userRepository, {
-    editorTools: editorToolsService,
+  const sharedDomainMethods: SharedDomainMethods = {
     /**
      * @todo find a way how to resolve circular dependency
      */
-  });
+    editorTools: editorToolsService,
+    noteSettings: noteSettingsService,
```

Comment:
its better to remove these changes if they are not used right now

Reply:
removed